### PR TITLE
Modify the existing shell commands to become more sudo friendly

### PR DESCRIPTION
**Pull request by Pouriya-Inmanta**
# Description

Modifies the existing shell commands to become more `sudo` friendly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# # v 1.3.1 (2021-08-22)
```
Comment (wouterdb) [inline CHANGELOG.md:1]: The changelog entry without date is unreleased, so you can add it there (or make it v1.2.2)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # v 1.3.0 (?)
 Changes in this release:
 
+# v 1.2.2 (2021-08-22)
```
Comment (wouterdb) [inline CHANGELOG.md:4]:
```suggestion
# v 1.2.2 (?)
```

@sanderr should any other file be updated to build release 1.2.2?

**Review by wouterdb — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
-# v 1.3.0 (?)
+# v 1.2.2 (?)
```
Comment (sanderr) [inline CHANGELOG.md:1]: So, as discussed on slack, I would undo this change. Keep the change entry, but revert the change to the version here.

**Review by sanderr — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (wouterdb) [reply CHANGELOG.md:1]: oeps, sorry, bad advice from my side.

**Review by wouterdb — APPROVED** [review]

**Review by sanderr — APPROVED** [review]
